pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -2871,7 +3220,7 @@ class WorldSessionActor extends Actor with MDCContextAware {
       import net.psforever.objects.GlobalDefinitions._
       import net.psforever.types.CertificationType._
       val faction = PlanetSideEmpire.VS
-      val avatar = Avatar(s"TestCharacter$sessionId", faction, CharacterGender.Female, 41, CharacterVoice.Voice1)
+      val avatar = new Avatar(41605313L+sessionId, s"TestCharacter$sessionId", faction, CharacterGender.Female, 41, CharacterVoice.Voice1)
```

Comment:
Uhhh, what does this magic number mean?

Reply:
This is part of my "dev branch stuffs," i.e., a local hack.  41605314 was the unique character identifier number for the original character we had encoded.  The first `sessionId` is typically 1.  On the live server, this whole identifier is a table row number.

Reply:
Yikes. I see. Can you add a comment about that or replace with a whole number to make it more clear?

Reply:
Since we're working to unify development and live, this sort of hack (in fact, all sorts of hack) will be completely retired eventually.  For now, I'll add the comment.

Reply:
The comment above that line has been expanded.